tests/conftest.py

```diff
@@ -27,6 +27,12 @@
 from sys import platform
 
 from _pytest.terminal import TerminalReporter  # for terminal customization
+
+from ansys.mapdl.core import _HAS_PYVISTA
+
+if _HAS_PYVISTA:
+    import ansys.tools.visualization_interface as viz_interface
```

Comment:
I rather you do this... but after ``has_dependency`` definition.
```suggestion
if has_dependency("pyvista"):
    import ansys.tools.visualization_interface as viz_interface
    viz_interface.TESTING_MODE = True
```